source/pmc/community-growth.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Community Growth
+tags: ["pmc","committers","community"]
+---
+
+Growing your project community takes work. This document makes practical
+suggestions of how to encourage people to get involved in your project.
+
+## Publish a road map
+
+Volunteers can work on whatever they want, and so we have a tendency to
+not want to tell them what to work on. However, publishing a roadmap, or
+even a wishlist of features or improvements that are desired, can give
+people an idea of where they might be able to get involved.
+
+This also gives a clear sense that the project has ambitions, and is
+going somewhere, rather than that it is stagnating.
+
+## Periodically publish the open issue list
+
+Your ticket tracker allows you to periodically summarize the open
```

Comment:
I would also to propose:
1. Tag some issues as "low hanging fruits" allowing new contributor to ramp up
2. For PR and issue, I would propose to send reminder to PRs/issues not recently updated (to avoid stale ones)
3. A good idea is also default assignee that can provide a first answer and add other people to help the contributor (mentoring process).